nuxeo-importer-core/src/main/java/org/nuxeo/ecm/platform/importer/base/GenericThreadedImportTask.java

```diff
@@ -233,6 +238,12 @@ protected DocumentModel doCreateLeafNode(DocumentModel parent, SourceNode node)
                 uploadedKO += fileSize;
             }
 
+            //
+            EventProducer service = Framework.getService(EventProducer.class);
+            EventContextImpl evctx = new DocumentEventContext(session, session.getPrincipal(),leaf);
```

Comment:
I prefer avoiding contractions, `eventContext` is fine. Also missing a space before `leaf`.